packages/react-on-rails-pro/package.json

```
@@ -7,7 +7,8 @@
     "build": "yarn run clean &&  yarn run tsc",
     "build-watch": "yarn run clean && yarn run tsc --watch",
     "clean": "rm -rf ./lib",
-    "test": "jest tests",
+    "test": "jest tests --testPathIgnorePatterns=\".*(ReactOnRailsRSC).*\"",
+    "test-rsc": "NODE_CONDITIONS=react-server jest tests/ReactOnRailsRSC.test.tsx",
```

```suggestion
    "test": "jest tests --testPathIgnorePatterns=\".*(ReactOnRailsRSC).*\" && NODE_CONDITIONS=react-server jest tests/ReactOnRailsRSC.test.tsx",
```
Or you can make separate `test:non-rsc` and `test:rsc` and call both from the `test` script. But with your current approach:
1. RSC tests won't run in CI;
2. Other developers need to remember to run `yarn run test-rsc` as well as `yarn run test`.

---

done